### ISSUE TITLE
Carthage Fails and can't use CocoaPods 

### ISSUE THREAD

**Issue by jjohnsonvng** [stephencelis/SQLiteCipher.swift#10]
**\* Checking out SQLiteCipher.swift at "aaf0b5107deb2006e6edd74d9ca3e362be240885"
**\* xcodebuild output can be found in /var/folders/0m/jz0qf0bd5834_n51gwwp73340000gn/T/carthage-xcodebuild.v4muZl.log
**\* Skipped building SQLiteCipher.swift due to the error:
Dependency "SQLiteCipher.swift" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/stephencelis/SQLiteCipher.swift/issues/new

**Comment by jjohnsonvng** [comment]
More Information
![screen shot 2016-03-24 at 8 56 54 pm](https://cloud.githubusercontent.com/assets/1311885/14036142/f0b348c2-f203-11e5-8490-a07c813ffe31.png)

**Comment by jjohnsonvng** [comment]
And a bit more
![screen shot 2016-03-24 at 9 04 27 pm](https://cloud.githubusercontent.com/assets/1311885/14036148/08b94b7e-f204-11e5-8f91-df849230f13e.png)

**Comment by jjohnsonvng** [comment]
The only way I got this to work successfully was to do a Manual Installation. Also NOTE: If you are also using SQLiteCipher you will have to Manually install it, but you will also have to copy the sqlite3.h file from SQLite into your project like show here.
![screen shot 2016-03-25 at 7 36 14 am](https://cloud.githubusercontent.com/assets/1311885/14044275/9515cfea-f25c-11e5-9bcc-7a99736cf95e.png)

You will then have to update the module.modulemap file in that same directory to this:
module SQLite3 [system] {
header "sqlite3.h"
link "sqlite3"
export *
}

Then everything works once again.

**Comment by gblotter** [comment]
@jjohnsonvng yeah... still not setting/getting that SQLite3 directory from the SQLite.swift submodule.  Don't know how to follow your example code when the code I get from the repo doesn't match ☹️

**Comment by jjohnsonvng** [comment]
I downloaded both SQLite and SQLiteCipher with GitHub

![screen shot 2016-03-25 at 11 23 07 am](https://cloud.githubusercontent.com/assets/1311885/14048399/fe040f7a-f27b-11e5-8574-6c545f988cff.png)

I could then see the /usr/include/sqlite3.h file

Regardless I understand your frustration trust me.

Here are the files

[Archive.zip](https://github.com/stephencelis/SQLiteCipher.swift/files/189724/Archive.zip)
